Formula/ansible@2.8.rb

```diff
@@ -593,7 +596,7 @@ class AnsibleAT28 < Formula
   end
 
   def install
-    ENV.prepend_path "PATH", Formula["python@3.8"].opt_libexec/"bin"
+    ENV.prepend_path "PATH", Formula["python"].opt_libexec/"bin"
```

Comment:
Could we use python@3.7 here @iMichka ?

Reply:
Does a python@3.7 formula exist?

Reply:
It's an alias for the python formula AFAIK.

Reply:
The alias does not exist right now, and will exist with #54912

I will migrate this to python@3.7 once we have fixed the migration issue we are encountering in  #54912. So please leave it as "python".